pyproject.toml

```diff
@@ -0,0 +1,84 @@
+[project]
+name = "pysiaf"
+description = "Handling of Science Instrument Aperture Files (SIAF) for space telescopes"
+requires-python = ">=3.8"
+authors = [
+    { name = "STScI", email = "help@stsci.edu" },
+]
+classifiers = [
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: BSD License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python",
+    "Topic :: Scientific/Engineering :: Astronomy",
+    "Topic :: Software Development :: Libraries :: Python Modules",
+]
+dependencies = [
+    "astropy>=4.3.1",
+    "lxml>=4.6.4",
+    "matplotlib>=3.4.3",
+    "numpy>=1.21.4",
+    "numpydoc>=1.1.0",
+    "openpyxl>=3.0.9",
+    "requests>=2.26.0",
+    "scipy>=1.7.2",
+]
+dynamic = [
+    "version",
+]
+
+[project.readme]
+file = "README.md"
+content-type = "text/markdown"
+
+[project.license]
+file = "LICENSE.md"
+content-type = "text/plain"
+
+[project.urls]
+documentation = "https://pysiaf.readthedocs.io/en/latest/"
+repository = "https://github.com/spacetelescope/pysiaf"
+
+[project.optional-dependencies]
+docs = [
+    "stsci_rtd_theme",
+    "sphinx_automodapi",
+    "numpy",
+    "matplotlib",
+    "scipy",
+    "tomli; python_version<\"3.11\"",
+]
+
+[build-system]
+requires = [
+    "setuptools>=61.2",
+    "setuptools_scm[toml]>=3.4",
+    "wheel",
+    "oldest-supported-numpy",
```

Comment:
Are you sure you need oldest-supported-numpy . Do you have C-extension that uses numpy ABI?

Reply:
@zacharyburnett I am assuming this a hold over from the `setup.py` file. I can double check and open a new PR with a fix.

Reply:
Looks like we versioned `numpy` in the `setup.py` file. Any reason for using the "oldest-supported-version" of numpy?

Reply:
This section is for *build*, not *install*. Do you need numpy during compile time?

Reply:
Ah, right sorry. I don't believe we do!